tests/conftest.py

```diff
@@ -189,8 +188,9 @@ def setup_eng(setup_backend_pars):  # pylint: disable=redefined-outer-name
     def _setup_eng(num_subsystems, **kwargs):
         """Factory function"""
         prog = Program(num_subsystems)
-        setup_backend_pars.update(kwargs)  # override defaults with kwargs
-        eng = Engine(backend=setup_backend_pars['backend_name'], **setup_backend_pars)
+        backend, backend_options = setup_backend_pars
+        backend_options.update(kwargs)  # override defaults with kwargs
+        eng = LocalEngine(backend=backend, backend_options=backend_options)
```

Comment:
At some point we'll have to make more of a distinction that this fixture only applies to LocalEngines. Any tests that look at the `state` can remain as-is. But we'll have to think about how to test other tests on remote engines

Reply:
Yep, I imagine almost all existing engine fixtures will be renames to be local engine fixtures, with new constructs required for remote engines :+1: